corehq/apps/hqcase/api/get_list.py

```diff
@@ -119,7 +135,12 @@ def _get_query(domain, params):
              .sort("@indexed_on")
              .sort("_id", reset_sort=False))
     for key, val in params.items():
-        query = query.filter(_get_filter(domain, key, val))
+        if isinstance(val, list):
```

Comment:
Why not move this logic to `_get_filter` where this fits better?

Reply:
+1 to pushing this down

Reply:
39289deae8f